sanic/blueprints.py

```diff
@@ -221,8 +221,12 @@ def static(self, uri, file_or_directory, *args, **kwargs):
         name = kwargs.pop('name', 'static')
         if not name.startswith(self.name + '.'):
             name = '{}.{}'.format(self.name, name)
-
         kwargs.update(name=name)
+
+        strict_slashes = kwargs.pop('strict_slashes', None)
```

Comment:
Can you use `get` instead of `pop` here? No reason to mutate the `kwargs` in this case.

Reply:
Fixed!